cmrtlib/README.md

```diff
@@ -0,0 +1,29 @@
+# Intel(R) C for Media Runtime
+
+
+## Introduction
+
+cmrtlib is a runtime library needed when user wants to execute their own GPU kernels on render engine. It calls iHD media driver to load the kernels and allocate the resources. It provides a set of APIs defined in agnostic/share/cm_rt.h for user to call directly from application.
+
+## License
+
+The Intel(R) Media Driver for VAAPI is distributed under the MIT license.
+You may obtain a copy of the License at:
+
+https://opensource.org/licenses/MIT
+
+## Building and Install
+
+cmrtlib is recommended to be built and installed together with media driver. Please refer to the readme in the root path.
+
+## Backward compatibility
+
+cmrtlib maintains the backward compatibility in both API level and ABI level
+
+## SDK package
+
+You can build your application with the SDK package we maintained in https://01.org/c-for-media-development-package/. In the SDK, we provided the CMake files to build your own application and kernels. You can also find the spec of APIs in the documents of the SDK package.
+
+## Notes
+
+This cmrtlib is a different project compared with https://github.com/intel/cmrt, although they have similar names and funtionalities. DO NOT replace one by the one.
```

Comment:
DO NOT replace one by **>>another<<**.

Reply:
This cmrtlib library is a separate effort from a similar library (https://github.com/intel/cmrt).  They may provide the same functionalities but this is not intended to replace the other. This cmrtlib works and is maintained to support the following platforms: (a link to driver's platforms maybe).